CHANGELOG.rst

```diff
@@ -7,6 +7,12 @@ foursight-smaht
 Change Log
 ----------
 
+0.1.0
```

Comment:
Higher in file than I can easily attach a comment, the ==='s are too short over and under foursight-smaht.